lib/ansible/modules/cloud/vmware/vmware_evc_mode.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Michael Tipton <mike () ibeta.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vmware_evc_mode
+short_description: Enable/Disable EVC mode on vCenter
+description:
+    - This module can be used to enable/disable EVC mode on vCenter.
+version_added: 2.9
+author:
+    - Michael Tipton (@castawayegr)
+notes:
+    - Tested on vSphere 6.7
+requirements:
+    - "python >= 2.6"
+    - PyVmomi
+options:
+  cluster_name:
```

Comment:
I have seen people using same cluster name across datacenters (same name clusters in development, production and QA datacenters) so it would be good idea to add datacenter parameter as well. That will help user to isolate correct cluster name from any given datacenter rather than relying on first found.